core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.ts

```diff
@@ -276,6 +279,18 @@ export class LessonInformationCardModalComponent extends ConfirmOrCancelModal {
     });
   }
 
+  returnToCheckpoint(index: number): void {
+    const cardIndex = this.checkpointCardsIndex[index];
+    if (cardIndex !== undefined) {
+      this.playerPositionService.setDisplayedCardIndex(cardIndex);
+      this.playerPositionService.onActiveCardChanged.emit();
+      // Close modal.
```

Comment:
Please treat comments as a last resort to explain code. Comments should not be added lightly, the main reason is that they are not the source of truth. The code will at some point diverge from the comment because somebody forgot to update it. This creates ambiguity and might lead to bugs, if somebody makes decisions based on outdated comments.

Here though, this comment is definitely not needed

Reply:
Comments removed